docs/examples/sentence_classification.md

```diff
@@ -0,0 +1,344 @@
+# Sentence classification
+
+In this tutorial we will try to predict whether a sms is spam or not. To train our model, we will use the `SMSSpam` dataset. This dataset is unbalanced, there is only 13.4% spam. Let's look at the data:
+
+
+```python
+from river import datasets
+
+datasets.SMSSpam()
+```
+
+
+
+
+    SMS Spam Collection dataset.
+    
+    The data contains 5,574 items and 1 feature (i.e. SMS body). Spam messages represent
+    13.4% of the dataset. The goal is to predict whether an SMS is a spam or not.
+    
+          Name  SMSSpam                                                                              
+          Task  Binary classification                                                                
+       Samples  5,574                                                                                
+      Features  1                                                                                    
+        Sparse  False                                                                                
+          Path  /Users/home/river_data/SMSSpam/SMSSpamCollection                                     
+           URL  https://archive.ics.uci.edu/ml/machine-learning-databases/00228/smsspamcollection.zip
+          Size  466.71 KB                                                                            
+    Downloaded  True                                                                                 
+
+
+
+
+```python
+from pprint import pprint
+
+X_y = datasets.SMSSpam()
+
+for x, y in X_y:
+    
```

Comment:
I know you well enough now to know that you like adding a lot of spaces between lines of code. It's a good reflex. However I feel that in this tutorial you went a bit far :). In this particular case I would say that there's no need for a space. But it's your notebook so your decision to make! :)